test/validate/pass_pointer_exists.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/sh
+
+set -o errexit
+set -o nounset
+
+TMP="$(mktemp -d)"
+cleanup() { rm -rf "$TMP"; }
+trap cleanup EXIT
+
+cat << 'EOF' > "$TMP/schema.json"
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "type": "object",
+  "properties": {
+    "nested": {
+      "type": "object",
+      "properties": {
+        "foo": { "type": "integer" }
+      },
+      "required": ["foo"]
+    }
+  }
+}
+EOF
+
+cat << 'EOF' > "$TMP/instance.json"
+{
+  "foo": 123
+}
+EOF
+
+if ! "$1" validate \
+    "$TMP/schema.json" \
+    "$TMP/instance.json" \
+    --path=/properties/nested \
```

Comment:
Instead of accessing a subschema of the schema, to make the test more representative, can you make the schema a very concise OpenAPI spec (in JSON) and access one of its schema components?